### ISSUE TITLE
My Home: Have 'Name your site' task direct users to the Customizer panel instead of Calypso Settings

### ISSUE THREAD

**Issue by jordesign** [Automattic/wp-calypso#53434]
Suggested change to the link in the "Name your site" CTA button.
We can get a win by directing users to use the Customizer for this - where they will see a preview of the change being made.

HOWEVER, a complicating factor in the current flow involved with that CTA: 
Name your site CTA --> Settings Page --> Wizard Help module --> Return to My Home. 
This needs thought/discussion before we decide to change the link.

ref: p1622831766261100-slack-C01VA100LEA

![image](https://user-images.githubusercontent.com/5665959/120946818-21666780-c781-11eb-8faa-8b7936aec665.png)

When the user clicks the "Name your site" CTA, the user it taken to:

CURRENT BEHAVIOR
The Calypso Site Settings page
![image](https://user-images.githubusercontent.com/5665959/120946899-61c5e580-c781-11eb-89ae-b8efe4bfbcad.png)

SUGGESTED IMPROVEMENT
Send the user to the Customizer.
![image](https://user-images.githubusercontent.com/5665959/120946759-efed9c00-c780-11eb-8798-81700ad445fd.png)

**Comment by jordesign** [comment]
May just be a case of redirecting the link on the button - but do we need to consider the tour?

<img width="1065" alt="Screen Shot 2021-06-06 at 7 17 58 am" src="https://user-images.githubusercontent.com/1842363/120905744-8ba2dd80-c697-11eb-98e5-8530fc5f6dfb.png">

**Comment by jordesign** [comment]
I've got a PR for the link change - but it doesn't address the need for the tour/wizard steps.
#53435

**Comment by autumnfjeld** [comment]
@jordesign  
> May just be a case of redirecting the link on the button - but do we need to consider the tour?

Good call out!!

We also need to consider the user flow here.
CURRENT:
My Home Checklist --> Name your site CTA --> Settings Page --> Wizard Help module --> Return to My Home. 

PROPOSED (not fleshed out, needs discussion):
My Home Checklist --> Name your site CTA --> Customizer --> Then what? --> Return to My Home?
(As you [pointed out in your PR](https://github.com/Automattic/wp-calypso/pull/53435/files#r646100808) )

**Comment by jeffgolenski** [comment]
@jordesign Thanks for logging this. 

As mentioned above by folks, the issue here isn't the quick change, it's the overall checklist experience. 

Options:

1) Current: Settings area redirect

**Pros**: CTA takes customer to settings, allows for name change, and brings them back to my home checklist. We have full control there so the guided tour isn't a problem and we can keep people on task. The quick accomplishment can help customers gain some momentum with the checklist.

**Cons:** Settings page doesn't have a big "wow factor" (I can't believe I just typed that) and doesn't promote extended feature discovery.

2) Proposed: Customizer redirect

**Pros**: Customizer allows people to see the initial preview of their site, which is important early on. 

**Cons:** Guided tour tasks are shaky here. May be a large mental hurdle for folks to parse the customizer, and likelihood of getting off task may be high.


**Other items to consider:**
* How could the proposed solution work for atomic sites (self hosted install)
* Would the customizer + site preview step be better suited for its own discovery task? The goal of this checklist task is to change the name of the site, and nothing else. Bloating that step may not be the best idea. 
* This task becomes less relevant for the newer wpcom onboarding flow which asks customer to name their site early on—if that onboarding flow becomes standard soon, is this worth investing much time in? I'd love to see data on the new onboarding flow to see.

Would love some thoughts from some of the other pod designers as well. News team assemble! @ollierozdarz @ianstewart 

If anything, I'd like to entertain a quick tech spike surrounding the guided tour within the customizer so the customers don't get lost.

**Comment by simison** [comment]
> This task becomes less relevant for the newer wpcom onboarding flow which asks customer to name their site early on—if that onboarding flow becomes standard soon, is this worth investing much time in? I'd love to see data on the new onboarding flow to see.

Also with FSE site name will be just directly on the editor canvas for them to modify, although anything we can do to help customers before that's reality is a plus.

**Comment by ianstewart** [comment]
> Pros [of settings page]: CTA takes customer to settings, allows for name change, and brings them back to my home checklist. We have full control there so the guided tour isn't a problem and we can keep people on task. The quick accomplishment can help customers gain some momentum with the checklist.

All these Pros seem possible with the existing Customizer tours and a redirect (assuming we can pass the correct value to the customizer and back to Calypso) on save from the customizer, back to the checklist?

> May be a large mental hurdle for folks to parse the customizer, and likelihood of getting off task may be high.

This feels like an even trade with the settings page. I'd argue it's harder to get off track from customization tasks in the customizer. The worst-case scenario is they explore tasks like …

- Updating their colour scheme.
- Getting a first look at where menus are (which they'll likely want to know about anyway).
- Perhaps tweak some homepage settings.
- etc.

There's less UI relative to the settings page and all (most?) the options provide a visual update.

> How could the proposed solution work for atomic sites (self hosted install)

It seems like an edge case that someone would have added a plugin before clicking on the first step on the checklist that tells them how to change the title of their site. If someone does that I'm convinced they don't need the checklist. :)

> Would the customizer + site preview step be better suited for its own discovery task? The goal of this checklist task is to change the name of the site, and nothing else. Bloating that step may not be the best idea.

I'd send them there to just change their Site Title.

**Comment by jeffgolenski** [comment]
> It seems like an edge case that someone would have added a plugin before clicking on the first step on the checklist that tells them how to change the title of their site. If someone does that I'm convinced they don't need the checklist. :)

@ianstewart Pardon my wordpress.com ignorance. I'm aware that sites are converted to atomic when a plugin is added, but aren't folks who purchase an e-commerce plan during onboarding automatically put on an atomic site?

**Comment by jordesign** [comment]
> Pardon my wordpress.com ignorance. I'm aware that sites are converted to atomic when a plugin is added, but aren't folks who purchase an e-commerce plan during onboarding automatically put on an atomic site?

This is true - they would automatically be Atomic. Looks like maybe 200-400 new subscribers a month from Scorecard.

**Comment by simison** [comment]
One more thing to consider is mobile experience — on settings there's no tour, nor we guide them back to my home:

<img width="377" alt="image" src="https://user-images.githubusercontent.com/87168/121156043-095e1900-c851-11eb-8cf6-03ab12aa160d.png">

On customizer they have the X little more visible, even when there would not be tour bringing them back:

<img width="379" alt="image" src="https://user-images.githubusercontent.com/87168/121156164-22ff6080-c851-11eb-9a34-6b40978940bc.png">

**Comment by ianstewart** [comment]
@jeffgolenski I forgot about the ecomm plan. Yep. That'd be correct in that case. They already get a dedicated onboarding in that plan though. Might be worth looking at how those two checklists interact.

**Comment by roo2** [comment]
I did some investigation of the tour capabilities of the customizer, currently we can only have a one step tour in the customizer, https://github.com/Automattic/wp-calypso/issues/54436#issuecomment-879012644

WRT taking the user back to my-home when they save the changes to the title, we are currently passing a return URL so that when the user hits the close button they're returned to my-home. I think that's actually better because the user may want to save their title multiple times or add a subtitle etc
